### ISSUE TITLE
Doesn't close windows with Zoom password as URL param

### ISSUE THREAD

**Issue by jbaruch** [andymckay/auto-close-zoom-tab#3]
URLs with https://zoom.us/j/$ID?pwd=$PWD don't match the current regex and won't close.

**Comment by grrttedwards** [comment]
If you are still interested in this extension, I've forked a fix over at https://github.com/grrttedwards/zoom-tab-closer

**Comment by jbaruch** [comment]
I am very much interested, thanks! I see no releases there, but I assume they are coming?

**Comment by grrttedwards** [comment]
Good catch, I should link back to the respective store pages for each browser in the README. 

It's published for Firefox here:

https://addons.mozilla.org/en-US/firefox/addon/zoom-tab-closer

And Chrome here:

https://chrome.google.com/webstore/detail/zoom-tab-closer/dipbdmjcnihnpjpignoilhljekeimlkp?hl=en
